### ISSUE TITLE
Color contrast

### ISSUE THREAD

**Issue by orangemug** [maputnik/editor#318]
The app has a few places where the contrast ratio is below standards, some include

 - Field labels with their default value 
 - The `expand` button

I'm sure there are also others. The work is to do a full review of the contrast ratios of elements in the app and fix them.

_Note:_ We're not looking for a redesign of the app just something in keeping with the current design.

**Comment by gregorywolanski** [comment]
This issue appears in the answers to question “What annoys you about Maputnik?” in the survey (original spelling):

- Difficult to read with black panel background, perhaps little lighter color to read layer names
